src/applications/hca/containers/IntroductionPage.jsx

```diff
@@ -13,12 +12,10 @@ import GetStarted from '../components/IntroductionPage/GetStarted';
 import content from '../locales/en/content.json';
 
 const IntroductionPage = ({ route }) => {
-  const { loading } = useSelector(selectEnrollmentStatus);
-  const { isUserLOA1 } = useSelector(selectAuthStatus);
-
-  useEffect(() => {
```

Comment:
This is not actually functioning, due to re-renders. A more global solution needs to be created on the `App.jsx` file to allow for a singular implementation that accounts for all re-renders.